### PR TITLE
New p5.js snippets

### PR DESCRIPTION
**Pull request by irealva**
Refining these snippets still...

### CODE REVIEW

**Comment by irealva** [comment]
@shiffman and @joeyklee I've added your proposed snippets for the image and audio libraries. Want to take a look?

There is one problem with the image one which is that the video is unflipped. Not sure how you'd like to handle this or if you could send us a PR for a better version. 

Also, we've updated our API and now have a more robust webcam class that exposes a canvas and makes the flipping/sizing super easy, the snippet here is updated: https://github.com/googlecreativelab/teachablemachine-community/tree/master/libraries/image

**Comment by irealva** [comment]
Also, would be great to get a pose snippet example in!

**Comment by joeyklee** [comment]
@irealva - you can pretty much ignore all of those comments since @shiffman has added those same notes here: https://github.com/googlecreativelab/teachablemachine-community/pull/21 

Re pose example:
+ We haven't yet integrated the teachable machine pose support into ml5. I will make a feature request for this now.

**Comment by shiffman** [comment]
Sorry for any confusion! @joeyklee `flipVideo()` is addressed in #21 if you want to take a look!

File: snippets/markdown/image/tensorflowjs/p5js.md [hunk 1]
```diff
@@ -1,59 +1,74 @@
...
+  
+  const size = 240;
+
+  // Load the model first
+  function preload() {
+    classifier = ml5.imageClassifier(imageModel + 'model.json');
+  }
+
+  function setup() {
+    const constraints = {
```
Comment (shiffman) [inline snippets/markdown/image/tensorflowjs/p5js.md]: I wonder if we can get away with skipping this, the flip is important, but this makes the example quite verbose and I have had no problems getting it to function correctly with a non square image?

**Comment by shiffman** [comment]
@googlebot I consent

File: snippets/markdown/audio/tensorflowjs/p5js.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
-```javascript
-// Global variable to store the classifier
+```html
+<div>Teachable Machine Audio Model - p5.js and ml5.js</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>
+<script src="https://unpkg.com/ml5@0.4.1/dist/ml5.min.js"></script>
```
[the remark this answers is not in the document]
Comment (irealva) [reply snippets/markdown/audio/tensorflowjs/p5js.md]: should we wait then before I merge this? Once i do it'll auto update TM2. so feel free to ping me once thats ready

File: snippets/markdown/audio/tensorflowjs/p5js.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
-```javascript
-// Global variable to store the classifier
+```html
+<div>Teachable Machine Audio Model - p5.js and ml5.js</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
```
[the remark this answers is not in the document]
Comment (irealva) [reply snippets/markdown/audio/tensorflowjs/p5js.md]: cool!

File: snippets/markdown/image/tensorflowjs/p5js.md (hunk 1 above)
Comment (irealva) [reply snippets/markdown/image/tensorflowjs/p5js.md]: Sure thing, i mean we can always change it later if people report any issues. 

So flip stays, constraints go away.

**Comment by irealva** [comment]
Reopened, we can merge tomorrow once the audio library is updated.

File: snippets/markdown/audio/tensorflowjs/p5js.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joeyklee) [reply snippets/markdown/audio/tensorflowjs/p5js.md]: I'm preparing the release now. I will ping you once everything is ready!

File: snippets/markdown/audio/tensorflowjs/p5js.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
-```javascript
-// Global variable to store the classifier
+Open up the code snippet below directly in the [p5.js Web Editor](https://editor.p5js.org/ml5/sketches/SoundModel_TM).
```
Comment (irealva) [inline snippets/markdown/audio/tensorflowjs/p5js.md]: Hey @joeyklee Im still seeing this sketch use the old model links:

```
let soundModelURL = 'https://storage.googleapis.com/tm-speech-commands/eye-test-sound-yining/model.json';
```

Can we change to the new model URL?

File: snippets/markdown/audio/tensorflowjs/p5js.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joeyklee) [reply snippets/markdown/audio/tensorflowjs/p5js.md]: Woohoo! ml5 `v0.4.2` is out: 

https://www.npmjs.com/package/ml5

```
<script src="https://unpkg.com/ml5@0.4.2/dist/ml5.min.js" type="text/javascript"></script>
```

File: snippets/markdown/audio/tensorflowjs/p5js.md (hunk 4 above)
Comment (joeyklee) [reply snippets/markdown/audio/tensorflowjs/p5js.md]: Just updated this! Should now be set to:

```
https://teachablemachine.withgoogle.com/models/h3p9R41J/model.json
```

**Comment by joeyklee** [comment]
@irealva  Alrighty, so from my end, I think everything is ready:
- [X] ml5 v0.4.2 is released and linked to our p5 web editor sketches
- [X] the model URLS have been updated on our examples and the p5 web editor.

**Comment by joeyklee** [comment]
@googlebot I consent

**Comment by joeyklee** [comment]
Woooo! 🌟

**Pull request merged**